### ISSUE TITLE
Error with code blocks

### ISSUE THREAD

**Issue by clemisch** [vinayak-mehta/present#51]
Thank you for this fun and interesting project.

I'm hitting an error when using inline code:

```
Hello `World`
```

leads to

```
Traceback (most recent call last):
  File "/home/clemisch/.local/lib/python3.6/site-packages/present/markdown.py", line 242, in parse
    Element = eval(element_name)
  File "<string>", line 1, in <module>
NameError: name 'Codespan' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/clemisch/.local/bin/present", line 8, in <module>
    sys.exit(cli())
  File "/home/clemisch/.local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/clemisch/.local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/clemisch/.local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/clemisch/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/clemisch/.local/lib/python3.6/site-packages/present/cli.py", line 18, in cli
    slides = markdown.parse(f.read())
  File "/home/clemisch/.local/lib/python3.6/site-packages/present/markdown.py", line 245, in parse
    f"(Slide {sliden + 1}) {element_name} is not supported"
ValueError: (Slide 1) Codespan is not supported
```

**Comment by clemisch** [comment]
Solved by installing newest `present` from github directly

**Comment by vinayak-mehta** [comment]
This should also be fixed by installing the latest version from PyPI: `pip install --upgrade present`.

**Comment by clemisch** [comment]
For me the PyPI version did not work. Maybe because I'm on Python 3.6 and it pulls an old version?

Btw, I commented out `python_requires=">=3.7"` in `setup.py` and everything works nicely. Is there a reason for 3.7 as minimum version?

**Comment by vinayak-mehta** [comment]
> Btw, I commented out python_requires=">=3.7" in setup.py and everything works nicely

Oh! I'll have to look into this then.

> Is there a reason for 3.7 as minimum version?

Yes, Python 3.7 got dataclasses in the standard library. dataclasses aren't available in Python 3.6 and below.

**Comment by vinayak-mehta** [comment]
> For me the PyPI version did not work. Maybe because I'm on Python 3.6 and it pulls an old version?

I think that it shouldn't even have pulled a version on old Pythons. https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

If you use Python 3.7 it'll work nicely without the need to install dataclasses using pip again.

**Comment by clemisch** [comment]
> I think that it shouldn't even have pulled a version on old Pythons

I don't think this is a big deal, but FYI it definitely _did_ pull `present` at version `0.4.0`. 

> dataclasses aren't available in Python 3.6 and below

Out of interest, is this the only reason for `python_requires=">=3.7"`? Is pulling (future) standard libraries as requirements bad practice?

**Comment by vinayak-mehta** [comment]
Yes dataclasses are the only reason for `python_requires=">=3.7"`.

> Is pulling (future) standard libraries as requirements bad practice?

I guess not, since the maintainer of the PyPI package is the maintainer of dataclasses in the standard library too.
